corehq/apps/locations/util.py

```diff
@@ -184,6 +185,7 @@ def _loc_type_dict(self, loc_type):
             uncategorized_keys.update(uncategorized_data.keys())
 
             loc_dict = {
+                'location_id': loc.location_id,
```

Comment:
Oh, and `consumption` is also in there regardless of whether or not it's used (the value is just `{}` if not relevant, 'cause that one IS expensive to compute).